src/main/kotlin/no/nav/k9/domene/modell/Kodeverk.kt

```diff
@@ -175,17 +175,45 @@ enum class BehandlingType(override val kode: String, override val navn: String,
 }
 
 @JsonFormat(shape = JsonFormat.Shape.OBJECT)
-enum class OppgaveKode(override val kode: String, override val navn: String) : Kodeverdi {
-    SYKDOM("9001", "Sykdom"),
-    INFOTRYGD_SØK("9007", "Infotrygdsøk"),
-    OMSORG("9020", "Omsorg"),
-    ETTERLYS_INNTEKTSMELDING("9068", "Etterlys inntektsmelding"),
-    AVKLAR_MANGLENDE_INNTEKTSMELDING("9069", "Avklar manglende inntektsmelding"),
-    FASTSETT_BEREGNINGSGRUNNLAG("5038", "Fastsett beregningsgrunnlag"),
-    MEDLEMSKAP("5053", "Medlemskap"),
-    FORESLÅ_VEDTAK("5015", "Forslå vedtak"),
-    FORESLÅ_VEDTAK_MANUELT("5028", "Foreslå vedtak manuelt"),
-    FATTE_VEDTAK("5016", "Fatte vedtak");
+enum class OppgaveKode(override val kode: String, override val navn: String, val gruppering: String) : Kodeverdi {
+    // Innledene behandling
+    MEDLEMSKAP("5053", "Medlemskap", "Innledende behandling"),
```

Comment:
grupperingen kunne ha vært en enum, siden det er en kode og for å redusere sjansen for stavefeil

Reply:
Godt poeng! Endrer til det